### ISSUE TITLE
(Un)subscribing crashes slide

### ISSUE THREAD

**Issue by MrGreenTea** [ccrama/Slide#343]
I tried to unsubscribe from r/calvinandhobbes through the sidebar and slide just crashed

Debug info:
 OS Version: 3.4.109-Resurrected_Kernel-g2e67f0b(89e81a6461)
 OS API Level: 22
 Device: A0001
 Model (and Product): A0001 (bacon)
 RELEASE: 5.1.1
 BRAND: oneplus
 DISPLAY: bacon-userdebug 5.1.1 LMY48W 89e81a6461 test-keys
 CPU_ABI: armeabi-v7a
 CPU_ABI2: armeabi
 HARDWARE: bacon
 MANUFACTURER: OnePlus
 SLIDE VERSION: 73

Stacktrace: 

```android.os.NetworkOnMainThreadException
	at android.os.StrictMode$AndroidBlockGuardPolicy.onNetwork(StrictMode.java:1147)
	at com.android.org.conscrypt.OpenSSLSocketImpl$SSLOutputStream.write(OpenSSLSocketImpl.java:749)
	at okio.Okio$1.write(Okio.java:78)
	at okio.AsyncTimeout$1.write(AsyncTimeout.java:155)
	at okio.RealBufferedSink.flush(RealBufferedSink.java:201)
	at com.squareup.okhttp.internal.spdy.Spdy3$Writer.synStream(Spdy3.java:339)
	at com.squareup.okhttp.internal.spdy.SpdyConnection.newStream(SpdyConnection.java:266)
	at com.squareup.okhttp.internal.spdy.SpdyConnection.newStream(SpdyConnection.java:242)
	at com.squareup.okhttp.internal.http.SpdyTransport.writeRequestHeaders(SpdyTransport.java:89)
	at com.squareup.okhttp.internal.http.HttpEngine$NetworkInterceptorChain.proceed(HttpEngine.java:814)
	at com.squareup.okhttp.internal.http.HttpEngine.readResponse(HttpEngine.java:684)
	at com.squareup.okhttp.Call.getResponse(Call.java:272)
	at com.squareup.okhttp.Call$ApplicationInterceptorChain.proceed(Call.java:228)
	at com.squareup.okhttp.Call.getResponseWithInterceptorChain(Call.java:199)
	at com.squareup.okhttp.Call.execute(Call.java:79)
	at net.dean.jraw.http.OkHttpAdapter.execute(OkHttpAdapter.java:64)
	at net.dean.jraw.http.RestClient.execute(RestClient.java:120)
	at net.dean.jraw.RedditClient.execute(RedditClient.java:147)
	at net.dean.jraw.RedditClient.execute(RedditClient.java:141)
	at net.dean.jraw.managers.AccountManager.setSubscribed(AccountManager.java:266)
	at net.dean.jraw.managers.AccountManager.subscribe(AccountManager.java:245)
	at me.ccrama.redditslide.Activities.SubredditOverview$13.onCheckedChanged(SubredditOverview.java:387)
	at android.widget.CompoundButton.setChecked(CompoundButton.java:154)
	at android.widget.CompoundButton.toggle(CompoundButton.java:113)
	at android.widget.CompoundButton.performClick(CompoundButton.java:118)
	at android.view.View$PerformClick.run(View.java:19899)
	at android.os.Handler.handleCallback(Handler.java:739)
	at android.os.Handler.dispatchMessage(Handler.java:95)
	at android.os.Looper.loop(Looper.java:135)
	at android.app.ActivityThread.main(ActivityThread.java:5310)
	at java.lang.reflect.Method.invoke(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:372)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:904)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:699)
```

**Comment by MrGreenTea** [comment]
Can't edit the original post, so here is some more info. 
I managed to reproduce this crash ever time I try to unsubscribe from any subreddit.

**Comment by ccrama** [comment]
Fixed. I was running the task on the main thread so the app crashes. Will fix it

**Comment by ccrama** [comment]
Fixed b88a383
